### DISCUSSION TITLE
Proposal for "Filecoin Hackathon Japan🚀

### DISCUSSION THREAD

**Discussion by KizukuTakara** [filecoin-project/community#532] [General]
Hi, Filecoin team.
My name is Takara.

I have a suggestion as I will take the initiative.
I am fascinated by Filecoin .

To expand the Filecoin ecosystem and further strengthen Filecoin's position indecentralized storage network, I propose to hold a "Filecoin Hackathon Japan".

*Temporary image
![Filecoin Hackathon Japan](https://user-images.githubusercontent.com/78465638/176366977-7472e879-10f8-41db-b31c-4e5853386a66.png)

As a premise, we believe that Filecoin will benefit from a hackathon in the following ways

1. Development of the Filecoin ecosystem through the development of related applications

2. Promotional effect by sharing Filecoin's technology and knowledge to users and builders.

In addition,there are three reasons why Filecoin needs to be held in Tokyo

1. The word "Web3" has become a national buzzword in Japan due to the comments of top entrepreneurs and celebrities.

2. It is the perfect time to involve Japanese engineers, the fourth largest group of engineers in the world, in your project.

3. Tokyo has world-class IP content such as manga and characters, as well as world-class game development capabilities, and there is potential for robust NFT and game content to be developed using file coin data storage.

As one of the world's hottest cities, Tokyo has already hosted two blockchain hackathons in the span of a month.

- Tokyo Hacker House | Solana
[https://solana.com/ja/events/tokyo-hacker-house]

- PolygonTokyoHack2022
[https://polygontokyohack.com/]

This is also an opportunity to create Japan's first Filecoin-based product.

By the way, Our team have connections with Japanese web3 entrepreneurs and influencers and can appeal to over 1000 Japanese web3 developers.

I would be happy to have anyone update me with any input, discussion, or ideas.

**Comment by web3jenks** [comment]
Hi @KizukuTakara I have brought this to PL & Filecoin Hackathon Operations team's attention now. Could you post this in our #hackathonathon in our Filecoin public slack as well?

**Comment by web3jenks** [comment]
Hi @KizukuTakara, would you be able to provide us with more information about your company/organisation and more details on your proposal? 

It is best if you could send us an email to our group email builders@filecoin.org to continue the conversation.

**Reply by KizukuTakara** [reply]
Thanks for giving me your email address.
In order to ensure smooth communication, would it be possible for you to provide us with the Discord ID or Twitter account of the person in charge?

**Reply by web3jenks** [reply]
Hi @KizukuTakara, we assess opportunities as a team. Please feel free to join our public slack workspace at https://filecoin.io/slack and discuss with the team direct at #hackathonathon channel. :-)
